bxml/buildingBlocks/forwardCall.md

```diff
@@ -0,0 +1,27 @@
+# Forward a Call
+
+## Concept
+
+To forward a call to another number, we will use the transfer BXML verb.
```

Comment:
`To forward a call to another number, we will use the [`<Transfer>`](../verbs/transfer.md) BXML verb.`

Reply:
Add links where possible